Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -83,15 +83,6 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
         }
     }
 
-    open var messageActionsForAlertMenu: [ChatMessageActionItem] {
```

Comment:
Why is this one removed?

Reply:
It did not make sense for the `ChatMessageActionsVC` to provide the actions for the AlertController in the `ChatMessageListVC`. The `ChatMessageActionsVC` is used for long pressing a normal message, and it shows the Popup view. When a message is bounced, it shows an alert controller which has nothing to do with the `ChatMessageActionsVC`. So this property was moved to the Message List with the name of `messageActions(forBouncedMessage:) -> [ChatMessageActionItem]`

Reply:
If this was used only for bounced messages, then it's ok (the method's name didn't clearly state that).

Reply:
Yeah it was used only for bounced messages. The previous name was indeed not clear.